src/main/java/com/locat/api/domain/endpoint/service/impl/PlatformEndpointServiceImpl.java

```diff
@@ -0,0 +1,73 @@
+package com.locat.api.domain.endpoint.service.impl;
+
+import com.locat.api.domain.endpoint.service.PlatformEndpointService;
+import com.locat.api.global.exception.ApiExceptionType;
+import com.locat.api.global.notification.NotificationException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import software.amazon.awssdk.services.sns.SnsClient;
+import software.amazon.awssdk.services.sns.model.*;
+
+import static com.locat.api.domain.user.entity.PlatformType.*;
+
+@Service
+@RequiredArgsConstructor
+public class PlatformEndpointServiceImpl implements PlatformEndpointService {
+
+    private final SnsClient snsClient;
+
+    @Value("${service.aws.sns.platform-application-arn.ios}")
+    private String iosArn;
+
+    @Value("${service.aws.sns.platform-application-arn.android}")
+    private String androidArn;
+
+    @Value("${service.aws.sns.topic-arn}")
+    private String topicArn;
+
+    private static final String PROTOCOL_APPLICATION = "application";
```

Comment:
![image](https://github.com/user-attachments/assets/ad18da9f-208d-43cd-9206-b09229c06373)
위 주석과 같이 여러 Protocol 중 저희가 사용할 기본 프로토콜을 정의하는 상수임을 명확히 알 수 있도록 개선해보는 건 어떨까요?
```suggestion
    private static final String DEFAULT_SNS_PROTOCOL = "application";
```